Internships Diagram.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE `internships` (
```

Comment:
Could we add foreign id to users (i.e., the user that posted the internship)? That way we can also show who posted the internship (was it instructor / student / etc).

Another thing to add would be time period, so that we can show when internship is. We can probably use enum or another table with trimester time periods (like T1/2023-2024 + TS/2023-2024).